spec.html

```diff
@@ -36662,16 +36657,16 @@ <h1>
       </emu-clause>
 
       <emu-clause id="sec-regexp.prototype-@@split">
-        <h1>RegExp.prototype [ @@split ] ( _string_, _limit_ )</h1>
+        <h1>RegExp.prototype [ @@split ] ( _string_, _substringLimit_ )</h1>
+        <p>This method finds substrings separated by matches of the regular expression *this* value in _string_ (ignoring its *"global"*, *"lastIndex"*, and *"sticky"* properties) and returns them in an Array with any match captures interleaved, truncated as necessary to satisfy _substringLimit_ (in which case later contents are absent rather than collected into an element that includes _separator_). A _substringLimit_ coerced to zero produces an empty Array, a *this* value that can match an empty String splits _string_ into individual code units (but will still also split on longer matches), and all other normal completion results have at least one element (which will be _string_ itself if no match is found).</p>
```

Comment:
> which will be _string_ itself if no match is found

Nit, but it will be the result of coercing `string` to a string. We probably don't need to capture that though.